bin/create-gutenberg-test-post.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Creates a post to test all Gutenberg blocks.
+ *
+ * @codeCoverageIgnore
+ * @package AMP
+ */
+
+/**
+ * Gets many of the Gutenberg fixture blocks in blocks/tests/fixtures/.
+ *
+ * @throws Exception If this is script is not run inside the plugin directory.
+ * @return string $content Post content with all Gutenberg blocks.
+ */
+function amp_get_blocks() {
```

Comment:
Since we're using PHP 5.3, let's just go ahead and use namespaces for this file. This will help prevent confusion with autocompletion. So then this could be renamed to just `get_test_data_blocks()`.